sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/SignatureAlgorithmTranslator.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Cryptography;
+
+namespace Azure.Security.KeyVault.Keys.Cryptography
+{
+    internal static class SignatureAlgorithmTranslator
```

Comment:
I would consolidate these on `SignatureAlgorithm` where we have a bunch of helpers already.